components/site-search/element.css

```diff
@@ -209,6 +209,10 @@
     color: inherit;
     background-color: var(--color-highlight-background);
   }
+
+  .divider {
+    color: var(--color-text-secondary);
+  }
```

Comment:
This doesn't seem to be used. Leftover from an earlier version?

```suggestion
```

Reply:
Not a leftover but a half-done styling clue. See latest version.